Forge.TreeWalker/src/TreeWalkerParameters.cs

```diff
@@ -51,10 +51,18 @@ public class TreeWalkerParameters
         public IForgeDictionary ForgeState { get; private set; }
 
         /// <summary>
+        /// [DEPRECATED]
         /// The ITreeWalkerCallbacks interface defines the callback Tasks that are awaited while walking the tree.
+        /// Prefers future callers to use the new ITreeWalkerCallbacksV2 CallbacksV2, and keep this ITreeWalkerCallbacks Callbacks for backward compatibility.
```

Comment:
>, and keep this ITreeWalkerCallbacks Callbacks for backward compatibility [](start = 84, length = 73)

Remove this part.

Reply:
removed

---
In reply to: [505911809](https://github.com/microsoft/Forge/pull/32#discussion_r505911809) [](ancestors = 505911809)